wordpress-mobile/WordPress-iOS
# Add Post title to Gutenberg editor

Fixes part of https://github.com/wordpress-mobile/gutenberg-mobile/issues/182

A title field is added to Gutenberg editor. This is a temporary field for alpha. Later we need to update it to make it scroll with the editor contents.

![simulator screen shot - iphone 7 - 2018-11-23 at 15 20 07](https://user-images.githubusercontent.com/5032900/48943456-52208180-ef34-11e8-8434-5fefa7106460.png)

**To test:** 

**Prerequisites:**

- run yarn start on your mobile-gutenberg path
- open XCode and Option+Click Run button
- Select "Run" option on the left
- Select Arguments on right and then check "gutenberg" option
- Click Run

**Test Steps**

- Open a post and Verify that its title is same with how it seems on post summary
- Update the title and save
- Close the post
- Verify that title is updated in the post summary
- Reopen the same post
- Verify that you see the updated title
